p4-16/spec/P4-16-draft-spec.mdk

```diff
@@ -1427,6 +1428,28 @@ Mpls_h[10] mpls;
 ~ End P4Example
 introduce a header stack called ```mpls``` containing 10 entries, each of type ```Mpls_h```.
 
+### Header Unions {#sec-header-unions}
```

Comment:
I think I was git-challenged today. I swear I added it but I don't see it here now or in my local repository. In any event, #190 looks good. Let's merge them both once somebody signs off on this one.